.neuro/live.yml

```diff
@@ -3,14 +3,14 @@ title: demo-oss-dogs
 id: mlops_demo_oss_dogs
 
 defaults:
-  preset: cpu-medium-p
+  preset: cpu-medium
   life_span: 1d
   env:
     # Those env vars will be assigned to every job in this flow
-    PACHY_URI: pachd.pachyderm.svc.cluster.local:30650
+    PACHY_URI: api.pachyderm.mlops.neu.ro:30650
     PACHY_REPO: dogs-demo
     PROJECT_GIT_REPO: git@github.com:neuro-inc/mlops-demo-oss-dogs.git
-    PROJECT_GIT_BRANCH: master
+    PROJECT_GIT_BRANCH: an/pachyderm-buckets-update
```

Comment:
what if `${{ git.branch }}` ?

Reply:
Either way it's confusing: you have to remember to do git push (to some branch) or pipeline will use old code/config.
I can change to `${{ git.branch }}` but that's not the root of the problem

Reply:
let's think about the problem root.
the issue is that someone might forget to do git push.
what if we raise a warning if `git diff` is not empty and `git` context is used?

Reply:
And also raise a warning if `git diff` is empty but parent branch is missing some commits from local. Also, there're other funny cases which we can't track (e.g. user has two remotes where I push to), etc.

Overall the idea of getting data from github seems incorrect. I think we should be supporting storage in this scenario, not git. This would be definitely out of scope for this PR of course

Reply:
you mean, storing code in GIT?

Reply:
yeah, that was done with 'reproducibility in mind'
and we achieved it via pushing/pulling code to/from GH
if instead, we raise a warning that your GH repo is out of sync, we could achieve 'reproducibility' with less pain.
sounds good, wdyt?

Reply:
I think we can't reliably find when something is out of sync. And by adding git to the equation we now want user to think about local + git + storage as sources of information that should be consistently in sync.

Reply:
Yes, if he wants 'reproducibility' - this should be done in any case.
Otherwise, the user will simply ignore those errors (or don't use git context).

For me, our current approach (pushing to GIT, and loading code into each job) is a bit more complex, than keeping in sync local<>storage with `upload()` and syncing git manually with the heuristic reminder.

Reply:
I can't imagine another approach, if you agree with the current proposition, let's create a FR in NF and we are good to go here.

Reply:
I don't understand, what FR in NF you want. Isn't `${{ git.branch }}` already supported?

Reply:
In case, if the user utilized `git` context and (1) `git status` is not clear (uncommitted changes / unpushed commits) - we print a warning.

Reply:
Ah, I see now! Created https://github.com/neuro-inc/neuro-flow/issues/705

Reply:
great, thank you!